template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+
+It’s normal to feel nervous about sharing your drafts, especially if you’re a new writer or if you don’t feel as confident in your subject matter knowledge yet.
+But your draft can only become the best template it can be if you invite and incorporate high quality feedback into your drafts.
+Successfully accepting advice on a draft is a key element that distinguishes expert writers from novice writers.
+
+Sharing your work with reviewers:
+
+- Allows you to see your draft with fresh eyes the way a new user would see it.
+- Can make you aware of key insights or perspectives that you hadn’t yet considered.
+- Can help you identify which parts of your draft need more careful thought, attention, and revision.
+
+As you receive feedback, try to give each comment the benefit of the doubt and seriously consider it.
+Sometimes new writers are tempted to react defensively to feedback on their work, but remember that your reviewers have the same goals that you have: to produce a high quality template!
+But also keep in mind that you don’t need to accept every suggestion.
+If you can make a good argument not to adopt a suggestion, that is important to consider as well.
+
+One other thing that might help you get more high quality reviews is to clearly indicate what kind of feedback you’re looking for, based on areas of the draft you think need some improvement. Do you need:
+
+- Global-level feedback, which includes advice on the big picture, general content, tone, clarity, and overall organization or flow of the document?
+- Local-level feedback, which includes wordsmithing paragraphs or sentences and polishing up the draft for final revision?
+
+Remember to be positive and show appreciation when people take time to review your drafts.
+Providing feedback takes time and energy.
+Treat each piece of feedback as a gift (even feedback that you possibly choose to disregard).
+Happy editing!
+
+
+## Submit a pull request
+
+The purpose of this phase is to ensure your template project meets the standards of the Good Docs Project and is ready for public distribution.
+In this phase, you’ll convert your template documents into Markdown and open a pull request in the `templates` repository on GitHub.
+
+If you are not comfortable working in Markdown, Git, or GitHub, a community member (such as your template mentor) will assist you.
+
+Once you’ve submitted a pull request, the templates coordinator will assign at least two members from the approved pull request reviewer list to review your template.
+These reviewers will check the template to ensure it meets the project’s formatting and quality guidelines as outlined in our template pull request checklist.
+Once the template has two approvals, it can be merged into the final project by a templates coordinator.
+
+
+## Improve the template with user feedback
+
+After you’ve submitted your pull request in the previous phase, your template will officially be part of the Good Docs Project. Yay!

```

Comment:
Is this true? Or are they officially part of the Good Docs Project after the PR is approved, rather than merely submitted?

Reply:
The way it's described is as if the pull request is a formality. Should we smash in the reviews here? 

I think the reason that it's organized this way is because it takes work to get the template from the googly doc to the pull request, so we don't want people to do the work, then have to go back to the drawing board, change it, and then update in the PR?

Reply:
Well, it's when it's merged, so I'll clarify that.

We want to give people the option to do a Google Doc (because it's easier to review) or markdown. It's up to the template author.

Reply:
I've adjusted some of the text here to add more clarification that this is a final quality check with experienced project leaders doing the final review. Resolving this comment thread.